catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1164,6 +1176,69 @@ public CreateResponse create(CreateRequest createRequest)
         return createResponse;
     }
 
+    private void applyAttributeOverridesToMetacardMap(Map<String, String> attributeOverrideMap,
+            Map<String, Metacard> metacardMap) {
+
+        if (MapUtils.isEmpty(attributeOverrideMap) || MapUtils.isEmpty(metacardMap)) {
+            return;
+        }
+
+        attributeOverrideMap.forEach((attributeMapKey, attributeMapValue) -> metacardMap.forEach((metacardMapKey, metacardMapValue) -> {
```

Comment:
I like the idea of avoiding the nested `forEach` and the readability of moving to a switch statement.  Pushed these changes after verifying everything still works as intended.